wagtail/contrib/wagtailsitemaps/sitemap_generator.py

```diff
@@ -9,7 +9,7 @@ def __init__(self, site=None):
         self.site = site
 
     def location(self, obj):
-        return obj.full_url
+        return obj.url
```

Comment:
Thanks for the clarification! In that case, this ought to be fixed so that it returns a local URL even on multi-site installations (where `obj.url` would return a full URL)... that's unrelated to the current PR, though, so I'll deal with that separately.